package.json

```diff
@@ -8,26 +8,28 @@
   "description": "What am I about?",
   "main": "server.js",
   "scripts": {
-    "start": "node server.js"
+    "start": "cross-env NODE_ENV=production node server.js",
```

Comment:
This could be simplified if we go with the comment in `fcctesting.js`:

`"start": "cross-env NODE_ENV=production node server.js",`